test/filetype/embeddedpuppet.vader

```diff
@@ -0,0 +1,22 @@
+Execute (Filetype detection on a new empty file):
+  edit foo.epp
+  AssertEqual &filetype, 'embeddedpuppet'
```

Comment:
These assertion args should be reversed - `<expected>, <got>`

Reply:
you're right, that's how vader's readme documents it. however its output is misleading. this is what it says when there's an error and it's in the documented order:

~~~
    (2/2) [   THEN] (X) 'embeddedpuppet' should be equal to 'mason'
~~~

but you're right, I'll change it for the documented order